heypublisher-sub-mgr.php

```diff
@@ -123,13 +120,7 @@
 
 // TODO: remove these defines and reference directly from XML code that makes query
 define('HEYPUB_SVC_URL_AUTHENTICATE','publishers/fetch_or_create');           # initial plugin authentication
-define('HEYPUB_SVC_URL_GET_PUBLISHER','publishers/show');                     # update the options
-define('HEYPUB_SVC_URL_UPDATE_PUBLISHER','publishers/update_publisher');      # update the options
-define('HEYPUB_SVC_URL_GET_GENRES','publishers/fetch_categories');            # fetch categories publisher accepts
-define('HEYPUB_SVC_URL_GET_PUB_TYPES','publishers/fetch_publisher_types');    # fetch publisher types
-define('HEYPUB_SVC_URL_GET_SUBMISSIONS','submissions/fetch_pending_submissions');           # fetch all pending submissions
 define('HEYPUB_SVC_URL_RESPOND_TO_SUBMISSION','submissions/submission_action');             # accept/reject/publish action
```

Comment:
This is supposed to be removed by #125 - where's the commit?

Reply:
Hmmmm - commit is .... lost??  Need to recover and apply....  This is still an issue.  The PUT needs to be going to the JSON endpoint.

Reply:
found and committed 57dc61f - this is now resolved